src/lib/translation/translations.ts

```diff
@@ -44,7 +46,9 @@ export const translations: I18n = {
 		'settings.discribtion': 'Here you can edit your game settings.',
 		'settings.theme.toggle': 'Automatic color scheme',
 		counter: 'Nacht',
-		state: 'state'
+		state: 'state',
+		'game.load': 'the first song is loading...',
```

Comment:
I would start the sentence with a Capital?

Reply:
the button labels are all Uppercase anyway